openedx_rocketchat/static/js/rocket_chat_discussion.js

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Since you are not using babel or any other compiler this sintax is only for browers which have support for es6, other browers will fail. You must specify which browers is not compatible with this sintax or use a compiler like babel to transcribe this es6 sintax to es5. @andrey-canon

Reply:
Applying the other requested changes, only the es5 syntax is used